### DISCUSSION TITLE
Can u provide the latest version ext-foreignobject.js

### DISCUSSION THREAD

**Discussion by echolove38** [SVG-Edit/svgedit#808] [New feature request]
I can find this file in 5.1.0 version, 
but can't find it in 7.x version,
will u provide it for me ?

**Comment by jfhenon** [comment]
The extension is in the archive folder by the way but it was not migrated to work with V7.
Ok, I will look into this in the next 2 weeks.

**Comment by echolove38** [comment]
> 

Thanks!

**Comment by echolove38** [comment]
> The extension is in the archive folder by the way but it was not migrated to work with V7. Ok, I will look into this in the next 2 weeks.

hi, lots of days passed , I still can't find the ext-foreignobject.js file, will u provide it ?

**Comment by SuperContraptionGuy** [comment]
I'm also poking around for foreignObject support.  I've modified the source to work with them without running over style tags and deleting children on a sanitize, but my code is a bit ugly.

**Comment by echolove38** [comment]
> 

the author said they will provide it , but now have not done

**Comment by jfhenon** [comment]
Sorry but this is not something we can do in a short time frame  because of other priorities.
You may try to attract developpers with https://bountysource.com 
A branch has been started but is not finished about this extension:
https://github.com/SVG-Edit/svgedit/tree/foreign-object

**Comment by echolove38** [comment]
> Sorry but this is not something we can do in a short time frame because of other priorities. You may try to attract developpers with https://bountysource.com A branch has been started but is not finished about this extension: https://github.com/SVG-Edit/svgedit/tree/foreign-object

ok , thanks for your reply
